client/my-sites/jetpack-search/jetpack-instant-search-config/additional.tsx

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
+import React, { ReactElement, Fragment } from 'react';
+import { translate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { Card } from '@automattic/components';
+import FormFieldset from 'calypso/components/forms/form-fieldset';
+import FormToggle from 'calypso/components/forms/form-toggle';
+import SettingsSectionHeader from 'calypso/my-sites/site-settings/settings-section-header';
+
+export default function JetpackSearchInstantSearchAdditionalConfig(): ReactElement {
+	return (
+		<Fragment>
+			<SettingsSectionHeader
+				id="jetpack-search-additional-settings"
+				showButton
+				title={ translate( 'Additional Settings', { context: 'Settings header' } ) }
+			/>
+			<Card>
+				<FormFieldset className="jetpack-instant-search-config__additional">
+					<FormToggle checked>{ translate( 'Show sort selector.' ) }</FormToggle>
+					<FormToggle checked>{ translate( 'Enable infinite scrolling.' ) }</FormToggle>
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **31** times:
`translate( 'Enable infinite scrolling' )` *ES Score: 12*